extensions/helix/package.build.ts

```diff
@@ -274,6 +274,230 @@ export const pkg = (modules: Builder.ParsedModule[]) => ({
+          },
+          "S": {
+            text: "Open workspace symbol picker",
+            command: "workbench.action.showAllSymbols",
+          },
+          "d": {
+            text: "Open diagnostic picker",
+            command: "workbench.action.problems.focus",
+          },
+          "D": {
+            text: "Open diagnostic picker",
+            command: "workbench.action.problems.focus",
+          },
+          "g": {
+            text: "Open changed file picker",
+            command: "workbench.view.scm",
+          },
+          "a": {
+            text: "Perform code action",
+            command: "editor.action.quickFix",
+          },
+          // "'": {
+          //   text: "Open last picker",
+          //   command: "", // TODO
+          // },
+          "G": {
+            text: "Debug",
+            command: "workbench.action.debug.start",
+          },
+          "w": {
+            text: "Window",
+            command: "dance.openMenu",
+            args: [{ menu: "window" }],
+          },
+          "y": {
+            text: "Yank selections to clipboard",
+            command: "dance.selections.saveText",
+            args: [{
+              register: "",
+            }],
+          },
+          "Y": {
+            text: "Yank main selection to clipboard",
+            command: "dance.run",
+            args: [{
+              code: [
+                "const editor = vscode.window.activeTextEditor;",
+                "if (!editor) {",
+                "  return;",
+                "}",
+                "const text = editor.document.getText(editor.selection);",
+                "await vscode.env.clipboard.writeText(text);",
+              ],
+            }],
+          },
+          "p": {
+            text: "Paste clipboard after selections",
+            command: "dance.edit.insert",
+            args: [{
+              handleNewLine: true,
+              where: "end",
+            }],
+          },
+          "P": {
+            text: "Paste clipboard before selections",
+            command: "dance.edit.insert",
+            args: [{
+              handleNewLine: true,
+              where: "start",
+            }],
+          },
+          "R": {
+            text: "Replace selections by clipborad content",
+            command: "editor.action.clipboardPasteAction",
+            args: [],
+          },
+          "/": {
+            text: "Global Search in workspace folder",
+            command: "workbench.action.findInFiles",
+          },
+          "k": {
+            text: "Show docs for item under cursor (hover)",
+            command: "editor.action.showHover",
+          },
+          "r": {
+            text: "Rename symbol",
+            command: "editor.action.rename",
+          },
+          "h": {
+            text: "Select symbol reference",
+            command: "editor.action.referenceSearch.trigger",
+          },
+        },
+
+        "window": {
```

Comment:
`window` is set in the `space` object, so it isn't actually added as a menu.

(plus nit, remove unnecessary quotes)

```suggestion

        window: {
```